### ISSUE TITLE
No piece of code for sending the email

### ISSUE THREAD

**Issue by Akashiutchiha** [mofirojean/Emailer_using_python#1]
I will like to handle the code for sending Emails. excel_reader python code will read the email address and the message from the Excel file and use the program i will write to send the mails.

**Comment by mofirojean** [comment]
its great feel free, i will be creating the template and the file to convert the templates into pdf

**Comment by mofirojean** [comment]
yes i made it in such a way we can take the information from a survey document  and reformat it to have only names and email addresses the we will use the information to insert the name in a template document, convert that document into a pdf and send it as an attachment to email address along with some customized message

**Comment by Akashiutchiha** [comment]
Did you had the idea giving this project a user interface?
Like we could decide to use pyscript together with html to produce even a website which uses python to send emails instead of php as usual.
